locations/storefinders/rio_seo.py

```diff
@@ -38,7 +38,14 @@ def parse_autocomplete(self, response: Response, **kwargs: Any) -> Any:
 
     def parse(self, response, **kwargs):
         map_list = response.json()["maplist"]
-        data = json.loads("[{}]".format(Selector(text=map_list).xpath("//div/text()").get()[:-1]))
+        try:
+            data = json.loads("[{}]".format(Selector(text=map_list).xpath("//div/text()").get()[:-1]))
+        except json.decoder.JSONDecodeError:
+            self.logger.warning("Could not parse response - check API output")
+            data = []
+        except TypeError:
```

Comment:
Perhaps just catch all exception types (along with `json.decoder.JSONDecodeError`) as they all appear to result in the outcome of "Could not parse response - check API output"